tee-worker/litentry/core/assertion-build/src/dynamic/mod.rs

```diff
@@ -68,14 +72,16 @@ pub fn build<
 				result.schema_url,
 				result.meet,
 			);
-			Ok(credential_unsigned)
+
+			let logs: Vec<String> = match params.return_log {
+				Some(true) => DYNAMIC_ASSERTION_LOGS.with(|v| v.borrow().iter().cloned().collect()),
```

Comment:
Is it possible to take ownership here instead of copying logs ?
Anyway, thread local variable should be cleaned up after every success/failure execution so there are no leftovers.

What do you think about returning logs from `execute_smart_contract` function ?